doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -566,78 +566,104 @@ The query returns all the entities that match all the matching conditions (AND l
-      with that exact name (temperature) and whose value is **not** that exact value (41).
+    + Single element, e.g. `temperature!=41`. For an entity to match, it must contain the target
+      property (temperature) and its value must **not** be the query value (41).
     + A list of comma-separated values, e.g. `color!=black,red`. For an entity to match, it must
-      contain an attribute with that name, and whose value is **not** any of the values in the list
-      (AND clause).
+      contain the target property, and its value is **not** any of the values
+      in the list (AND clause) (or not include **any** of the values in the list in the case the target
+      property value is an array).
       Eg. entities whose attribute `color` is set to `black` will not match, while entities whose
       attribute `color` is set to `white` will match.
     + A range, specified as a minimum and maximum separated by `..`, e.g. `temperature!=10..20`.
-      For an entity to match, it must contain an attribute with that exact name (temperature) and
-      whose value is **not** in the interval between the minimum and the maximum (both included).
-      Ranges can only be used with attributes that represent dates (in ISO8601 format), numbers or
-      strings.
+      For an entity to match, it must contain target property (temperature) and its value must **not**
+      be in the interval between the minimum and the maximum (both included). Ranges can only be used
+      with elements target properties that represent dates (in ISO8601 format), numbers or strings.
 + **Greater than**: `>`. The right-hand side must be a single element, e.g. `temperature>42`.
-    For an entity to match, it must contain an attribute with that exact name (temperature) and
-    whose value is strictly greater than that value (42).
-    This operation is only valid for attributes of type date, numbers or strings (used with
-    attributes of other types may lead to unpredictable results).
+    For an entity to match, it must contain the target property (temperature) and
+    its value must be strictly greater than that query value (42).
+    This operation is only valid for target properties of type date, numbers or strings (used with
+    target properties of other types may lead to unpredictable results).
 + **Less than**: `<`. The right-hand side must be a single element, e.g. `temperature<43`.
-    For an entity to match, it must contain an attribute with that exact name (temperature) and
-    whose value is strictly less than that value (43).
-    This operation is only valid for attributes of type date, numbers or strings (used with
-    attributes of other types may lead to unpredictable results).
+    For an entity to match, it must contain the target property (temperature) and
+    its value must be strictly less than that value (43).
+    This operation is only valid for target properties of type date, numbers or strings (used with
```

Comment:
Actually, that refers to a JSON data type, not to the default NGSI that is set when no explicit type is provided. We cannot refer here to DateTime, Number, Text, etc. Note that a given user could use those literals even if the JSON data type is other, eg:

```
POST /v2/entities
{
  "id": "E",
  "A":  {
    "type": "Boolean",
    "value": 5
   }
}
```

As much, we can change the wording to mean "JSON type":

This operation is only valid for target properties of JSON type ...